src/sentry/static/sentry/app/data/platforms.jsx

```diff
@@ -1,3 +1,4 @@
+/* eslint import/no-unresolved:0 import/order:0 */
```

Comment:
If we don't do this then we would need to install python deps to generate the integration docs.